bin/generateChromeManifest.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python
+
+##
+# This script is designed to generate the chrome.manifest for the add-on in
+# the propert format, taking into account the locales present in the repo.
+##
+
+from __future__ import print_function
+
+import argparse
+import io
+import os
+import re
+
+# defaults
+DEF_L10N_SRC = os.path.join("locale")
+DEF_INPUT_FILE_NAME = os.path.join("add-on", os.extsep.join(["jar", "mn"]))
+DEF_OUTPUT_FILE_NAME = os.path.join("built", "add-on", os.extsep.join(["chrome", "manifest"]))
+
+
+def main(l10n_src, input_file_name, output_file_name):
+    print("Using l10n tree from", l10n_src)
+
+    locale_dirs = sorted(os.listdir(l10n_src))
+
+    input_file = io.open(input_file_name, "r")
+
+    jar_mn = input_file.read()
+
+    # sed -n -e '/^%/p' $(ADD-ON)/jar.mn | \
+    # sed -e "s/^% //g" | sed -e "s/%/chrome\//g" \
+
+    # Drop any line that doesn't begin with %.
+    manifest_output = "\n".join(line for line in jar_mn.split("\n") if (re.match("^%", line)))
+
+    # Drop the "% " from the start of lines.
+    manifest_output = re.sub(
+        '^% (.*$)',
+        '\\1',
+        manifest_output, 0, re.M)
+
+    # Replace remaining % with "chrome/".
+    manifest_output = re.sub(
+        '%',
+        'chrome/',
+        manifest_output, 0)
+
+    localeLines = ['locale loop {0} chrome/locale/{0}/'.format(x) for x in locale_dirs]
+
+    manifest_output = re.sub(
+        '(locale loop .*\n)',
+        '\n'.join(localeLines) + "\n",
+        manifest_output)
+
+    # For the loop-locale-fallback we need to adjust the path for inclusion.
+    # Mozilla-central actually duplicates file, the add-on by itself doesn't.
+    manifest_output = re.sub(
+        '(chrome/content/locale/en-US/)',
+        'chrome/locale/en-US/',
+        manifest_output)
+
+    print (manifest_output)
```

Comment:
Was this for debugging? Or do we want it to show up when building chrome.manifest?